sdk/cosmos/azure-cosmos-spark_3-4_2-12/README.md

```diff
@@ -157,6 +160,7 @@ https://github.com/Azure/azure-sdk-for-java/issues/new
 #### azure-cosmos-spark_3-1_2-12
 | Connector    | Supported Spark Versions | Supported JVM Versions | Supported Scala Versions | Supported Databricks Runtimes |
 |--------------|--------------------------|------------------------|--------------------------|-------------------------------|
+| 4.36.1       | 3.2.0 - 3.2.1            | [8, 11]                | 2.12                     | 10.\*                         |
```

Comment:
There seems to be a mismatch in `Supported Databricks Runtimes` across READMEs for `azure-cosmos-spark_3-1_2-12`?

Reply:
Fixed